packages/playwright/src/generators/configuration/configuration.ts

```diff
@@ -97,7 +97,12 @@ export async function configurationGeneratorInternal(
     if (isTsSolutionSetup) {
       // skip eslint from typechecking since it extends from root file that is outside rootDir
       if (options.linter === 'eslint') {
-        tsconfig.exclude = ['dist', 'eslint.config.js'];
+        tsconfig.exclude = [
```

Comment:
I realized that PW didn't exclude the same files as our other generators, so this updates them. It'll also help once we rename `eslint.config.js` to `eslint.config.cjs` to resolve CJS/ESM issues.